platform/migration/platform-migratable-api/pom.xml

```diff
@@ -48,15 +73,18 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Export-Package>org.codice.ddf.migration</Export-Package>
+                        <Export-Package>
+                            org.codice.ddf.migration,
+                            org.codice.ddf.util.function
```

Comment:
❓ Should we wait until those functions have been moved to a common module and officially reviewed outside this PR before exporting them?

Reply:
They have to be exported since they are part of the interface.